src/timer.rs

```diff
@@ -52,9 +52,9 @@ impl Timer {
 
     pub fn do_cycle(&mut self, ticks: u32) {
         self.internaldiv += ticks;
-        while self.internaldiv >= 256 {
+        while self.internaldiv >= 1024 { //256*4
```

Comment:
My bad, it's wrong there, it's 256, not 1024.

the div internal clock increment at 16384 Hz -> 1/16384 = 61035 nanosecondes.
clock cycle is 238 ns, so 61035 / 238 = 256 clock cycles.